README.md

```diff
@@ -57,6 +57,11 @@ Run the script below and replace the following parameters:
   * `GOOGLE_CLOUD_REGION`: You must choose a geographic region for Cloud Dataflow
   to process your data, for example: `us-west1`. For more information please refer to
   [Setting Regions](docs/setting_region.md).
+  * `GOOGLE_CLOUD_LOCATION`: You must choose a geographic location for Cloud Life
+  Sciences API to orchestrate job from. This is not where the data will be processed,
+  but where some operation metadata will be. This can be the same or different from
+  the region chosen for Cloud Dataflow. See the list of
+  [Currently Available Locations](https://cloud.google.com/life-sciences/docs/concepts/locations)
```

Comment:
I am thinking to keep our docs simple by only including required flags; all optional flags (location, service account, subnetwork, private IPs) will be explained in the `setting_region.md` page.
What do you think?

Reply:
I think that's fine for the other flags, but I think that location should stay here. Even though it's not technically required, I think it's good practice for users to explicitly specify this and to avoid potential questions if people are looking at the Cloud Life Sciences API operations and can't find the operation. By being clear up front about location it provides more info for users to control, find, and troubleshoot this.

Reply:
SG, thanks for explanation.

Let's fix this sentence then: "but where some operation metadata will be [stored?]."  Please also update the other 2 occurrences.

Reply:
Updated. Also changed "must" to "may" and added a note that the default is us-central1.